openedx/core/djangoapps/site_configuration/management/commands/create_or_update_site_configuration.py

```diff
@@ -0,0 +1,114 @@
+"""
```

Comment:
Can you keep this management command named `create_site_configuration` and add some logs about the behavior (updates values if `SiteConfiguration` already exists).
This is to prevent renaming this [here](https://github.com/edx/configuration/blob/3819c5245d22152d7630972e7d4e85334927665f/playbooks/masters_sandbox.yml#L40) and possibly on more places.

Reply:
@giovannicimolin, I renamed the command because we already have management commands named like `create_or_update_site` and that makes the behaviour obvious. Also, I have already created a [PR](https://github.com/edx/configuration/pull/5504) for the `configuration` repository changes and made this depend on that. So I still think we should keep the name change in this PR as-is.